web/settings/modulconfigevu.php

```diff
@@ -698,13 +698,22 @@ function getCookie(cname) {
 								<div class="col">
 									<input class="form-control" type="text" pattern="^((\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.){3}(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])$" name="solaredgeip" id="solaredgeip" value="<?php echo $solaredgeipold ?>">
 									<span class="form-text small">
-										Gültige Werte IP.<br>
+										IP Adresse des Solaredge Wechselrichters im lokalen Netzwerk.<br>
+									</span>
+								</div>
+							</div>
+							<div class="form-row mb-1">
+								<label for="solaredgemodbusport" class="col-md-4 col-form-label">IP Adresse</label>
+								<div class="col">
+									<input class="form-control" type="number" min="1" step="1" name="solaredgemodbusport" id="solaredgemodbusport" value="<?php echo (empty($solaredgemodbusport)?'502':$solaredgemodbusport) ?>">
```

Comment:
Die neue Einstellung `solaredgemodbusport` muss in der Datei `openWB/runs/updateConfig.sh` mit einem Standardwert in der openwb.conf ergänzt werden.

Reply:
Habe ich ergänzt und den Pull Request erweitert.